### ISSUE TITLE
Files cannot be opened in editor

### ISSUE THREAD

**Issue by mloetkemann** [codenvy/codenvy#2532]
Sometimes project files cannot be opened in editor. When I select a file to open it, nothing happens. Unfortunately, there is no error message. Also I cannot find any similiarities between those files.

**Codenvy version:**  Cloud 5.20.0

**Comment by eivantsov** [comment]
@mloetkemann are there any errors in browser dev console?

**Comment by mloetkemann** [comment]
Unfortuenately, I cannot see any errors. But I can see that file content is loaded.

**Comment by eivantsov** [comment]
@mloetkemann unfortunately, there's little information to attempt to reproduce this issue. It's might be a networking issue though

**Comment by mloetkemann** [comment]
@eivantsov Yes I know. Sorry for that. I have checked console again. Now, I see an error happen during startup (for example after refreshing with F5). I don't know if this is related to that. 

Please see following screenshot of console error:
![codenvy_error](https://user-images.githubusercontent.com/8386555/33186655-5843e476-d08b-11e7-8bcc-154067bb7d4b.png)

**Comment by mloetkemann** [comment]
It seems that directly when I create a new file, this file can never be opened

**Comment by mloetkemann** [comment]
@eivantsov: I think I have found how to replicate this issue! It seems that every file which starts with an empty line cannot be loaded into the editor. Means, that these files cannot be edited. Can you please check this?

Issue still exists on 5.22.0

**Comment by eivantsov** [comment]
@mloetkemann cannot reproduce using `eclipse/che:nightly`

**Comment by shawnr** [comment]
I still see this problem. Is there any progress on it? It is easy to replicate by creating a new file in the project and then trying to open it. It will commit and push the file no problem, but it will not open the file for editing. I see the exact same error mentioned above. I have been using Codenvy with a group of people and everyone has been able to duplicate this.

**Comment by shawnr** [comment]
![codenvy-file-open-bug](https://user-images.githubusercontent.com/38453/38513534-5be8e374-3be3-11e8-8095-f03d68bb3800.gif)
Here is a LiceCap of the bug in action.

**Comment by mloetkemann** [comment]
It is still mit working on my side.

>

**Comment by shawnr** [comment]
I have confirmed that this issue does not exist in the vanilla version of Eclipse Che, but it still exists on Codenvy. However, I have also found a workaround: When making a new file if you choose New > HTML File (or whatever kind of file), it will create a new file and populate it with some boilerplate code. This allows the editor to both create and open the file, which can then be edited. Once the file contains content, it can be renamed. See the animation below. /cc @terrywbrady

![new-file-workaround](https://user-images.githubusercontent.com/38453/39265639-a4d054aa-487c-11e8-985c-75edb372c414.gif)
